lib/src/component_declaration/component_base.dart

```diff
@@ -693,6 +693,10 @@ abstract class UiProps extends MapBase
   void disableRequiredPropValidation() {
     _shouldValidateRequiredProps = false;
   }
+
+  /// Names of props to opt out of required prop validation for.
```

Comment:
#nit Could be helpful to provide maintainers with a little more context
```suggestion
  /// Names of props to opt out of required prop validation for.
  ///
  /// Overridden in generated code, based on the value of `@Props(ignoreRequiredProps: ...)`.
```